blathering-barnacles/Bangazon
# #2 Richard L HR should be able to add an employee

Description:

Added the ability to add an employee to the employee list. Steps completed:  Added new URLPattern that takes the user to the addEmployee_view when the user clicks the appropriate hyperlink on the Employee List template. Created new addEmployee template.  Template includes a form that accepts all fields required for the Employee table. Created new addEmployee_view.  View includes new addEmployee method which takes information sent from the template and post a new employee to the employee table. Added link on employeeList template to take the user to the Add Employee form stored within the addEmployee template. 

Steps to Test:

go to localhost:8000/workforce/employees
Click the “Add New Employee” link
Fill out the new Employee form and click the Save button
After the application is rerouted to the Employee List page, verify that the new Employee appears in the list of employees.

Link to Feature Ticket:

#2 HR should be able to add an employee [https://github.com/blathering-barnacles/Bangazon/issues/2](url)
